node/cmd/ccq/http.go

```diff
@@ -37,6 +37,17 @@ type httpServer struct {
 }
 
 func (s *httpServer) handleQuery(w http.ResponseWriter, r *http.Request) {
+	// Set CORS headers for the preflight request
+	if r.Method == http.MethodOptions {
+		w.Header().Set("Access-Control-Allow-Origin", "*")
```

Comment:
In the cloud functions this line is before the **if** checking for options.  Will defer to whatever @evan-gray  thinks.